docs/conf.py

```diff
@@ -49,7 +49,7 @@
 author = 'The Trieste Contributors'
 
 # The full version, including alpha/beta/rc tags
-release = 'v0.1.0'
+release = 'v0.2.0'
```

Comment:
we should move to a VERSION file so we don't forget to update both versions, but I'm not going to do that now